src/components/Pagination/Pagination.stories.tsx

```diff
@@ -108,7 +108,6 @@ const paginationArgs: Object = {
     PaginationLayoutOptions.Next,
   ],
   pageSize: 10,
-  pageSizes: [10, 20, 30, 40, 50, 100],
```

Comment:
Same as above. The component functionality needs to remain the same, but with your fix as an improvement.

Reply:
Reverted this change